File: TRIANGLE PATTERN 1/code2.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter your no. of rows : ");
	scanf("%d",&row);

	for(int i = 1; i <= row ; i++){

		for(int j = 0 ; j < i ;j++){

			printf("%c\t",96 + i - j );
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code2.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		for(int i = 0; i < row ; i++){

			for(int j = 0; j < row ; j++){

				if(i % 2 == 0){

					System.out.print((char)(64+row-j)+"		");
				}else{
					System.out.print((char)(65+j)+"		");

				}

			}

				System.out.print("\n\n");

		}

	}

}
